pyproject.toml

```diff
@@ -56,6 +56,9 @@ build-backend = "setuptools.build_meta"
 
 [project.optional-dependencies]
 soap-bpnn = []
+alchemical-model = [
+  "torch_alchemical @ git+https://github.com/abmazitov/torch_alchemical.git",
```

Comment:
if/when we want to release metatensor-models on PyPI, all dependencies will also need to be on PyPI. I don't know what are your long term plans with torch_alchemical, but would it make sense to have the corresponding code in this repo instead of a separate one? Ideally by implementing all the modules in torch_alchemical in terms of metatensor-learn/metatensor-operations

I understand this might not be possible right now, so this is not to be done in this PR!

Reply:
Yeah that definitely needs to be discussed